astropy/table/sorted_array.py

```diff
@@ -0,0 +1,286 @@
+import numpy as np
+
+class SortedArray(object):
+    '''
+    Implements a sorted array container using
+    a list of numpy arrays.
+
+    Parameters
+    ----------
+    lines : `Table`
```

Comment:
It's confusing to me to call this `lines` because that implies something like a list of lines (rows).  Maybe `index_table` or even just `table`?